syntax = "proto3";

package tensors;

message NumTensor {
  repeated NumMatrix my_arrays = 1;
}

message NumMatrix {
  repeated NumRow my_sub_arrays = 1;
}

message NumRow {
  repeated int32 my_array = 1 [packed=true];
}